codalab/lib/bundle_cli.py

```diff
@@ -1857,10 +1857,25 @@ def do_wls_command(self, args):
         ),
     )
     def do_wrm_command(self, args):
+        delete_current = False
         for worksheet_spec in args.worksheet_spec:
             client, worksheet_uuid = self.parse_client_worksheet_uuid(worksheet_spec)
+            if (client, worksheet_uuid) == self.manager.get_current_worksheet_uuid():
+                delete_current = True
             client.delete_worksheet(worksheet_uuid, args.force)
 
+        if delete_current:
+            # Go to home worksheet
```

Comment:
We can, but it would involve artificially creating an `args` object. What do you think?